### ISSUE TITLE
Allow to customize length of VARCHAR in field definitions

### ISSUE THREAD

**Issue by danaki** [acelaya/doctrine-enum-type#28]
I have the following snippet that doesn't work, e.g. length is ignored:
```
     * @ORM\Column(type=PaymentDirection::class, nullable=false, length=10)
```

Resulting field is always VARCHAR(255) in MySQL. You can easily fix this line https://github.com/acelaya/doctrine-enum-type/blob/master/src/Type/PhpEnumType.php#L43
by making use of $fieldDeclaration like:
```
    public function getSQLDeclaration(array $fieldDeclaration, AbstractPlatform $platform)
    {
        return $platform->getVarcharTypeDeclarationSQL($fieldDeclaration);
    }
```

Do you need a PR for that?

**Comment by acelaya** [comment]
Yep, it makes sense. A PR would be really appreciated :)

**Comment by danaki** [comment]
#29

**Comment by acelaya** [comment]
Just released v2.2.2 including this bugfix.

Thanks for contributing!
